### PR TITLE
Add docs to show how to select specific simulator.

### PR DESCRIPTION
**Pull request by joelcloralt**
Add a message to let people know they can use the `--simulator` flag to run their apps on different simulators instead of the default "iPhone 6"

### CODE REVIEW

**Comment by facebook-github-bot-7** [comment]
By analyzing the blame information on this pull request, we identified @JoelMarcey and @hramos to be potential reviewers.

**Comment by facebook-github-bot-7** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by hramos** [comment]
Thanks for the pull request! After reviewing it, I am not sure if the Getting Started is the best place to document this. I believe the Getting Started guide should list the bare minimum steps to get up and running.

Last week I removed an entire section from the Android guide on using Genymotion and other emulators, and the plan is to document that kind of tips and tricks on a separate document. Perhaps a new article under the iOS Guides section might work better?

**Comment by mkonicek** [comment]
Thanks for helping us improve the docs! I agree with @hramos we should keep the Getting Started guide simple.

File: docs/QuickStart-GettingStarted.md [hunk 1]
```diff
@@ -195,6 +195,10 @@ react-native run-ios
 
 If everything is set up correctly, you should see your new app running in the iOS Simulator shortly.
 
+> You can specify the device the simulator should run with the `--simulator` flag, followed by the device name as a string. The default is `"iPhone 6"`. Example: `react-natvie run-ios --simulator "iPhone 4s"`.
```
Comment (mkonicek) [inline docs/QuickStart-GettingStarted.md]: typo: react-natvie

**Comment by joelcloralt** [comment]
Thanks guys, will update this PR

**Comment by JoelMarcey** [comment]
@joelcloralt Did you sign the CLA? We don't have a record of it.

Also, can you please rebase this as well?

Thanks!

File: docs/QuickStart-GettingStarted.md [hunk 2]
```diff
@@ -195,6 +195,10 @@ react-native run-ios
...
+> The device names corresponds to the list of devices available in xcode. You can check your available devices by running `xcrun simctl list devices` from the console.
```
Comment (hramos) [inline docs/QuickStart-GettingStarted.md]: Xcode

**Comment by facebook-github-bot-9** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by joelcloralt** [comment]
@JoelMarcey just completed it. Sorry about that. I updated the PR to have the new content in the navigation under Guides iOS instead of the getting started guide.

**Comment by JoelMarcey** [comment]
@joelcloralt Great. Thanks. This seems reasonable. Can you just remove the extraneous space addition that was made in Getting Started and then we can ship it.

File: docs/QuickStart-GettingStarted.md [hunk 3]
```diff
@@ -195,6 +195,7 @@ react-native run-ios
 
 If everything is set up correctly, you should see your new app running in the iOS Simulator shortly.
 
+
```
Comment (JoelMarcey) [inline docs/QuickStart-GettingStarted.md]: Can we remove the extraneous blank line entry here?

**Comment by mkonicek** [comment]
Looks cool @joelcloralt! Thanks for reviewing @JoelMarcey.

**Comment by mkonicek** [comment]
Please update the labels, it makes it easier to find PRs which are stuck in review. But what's up with the bot updating the labels?

**Comment by JoelMarcey** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**